juniper_official/Solutions/EVPN-VXLAN/check-irb-mac-count-netconf.rule

```diff
@@ -0,0 +1,96 @@
+/*
+ * Collects IRB unique MAC address count periodically and notifies anomalies  
+ * when MAC count is above static threshold.
+ *
+ * One input control detection
+ * 
+ *   1) threshold, is the threshold that causes the rule to report an anomaly. 
+ *      By default default it's 512. This rule will set a dashboard color to
+ *      red when mac address count is greater than token-threshold. Otherwise
+ *      color is set to green.
+ */
```

Comment:
s/One input control detection/One input controls detection/g
s/threshold,/threshold variable,/g

Reply:
updated the rule